src/config/middlewares.js

```diff
@@ -1,51 +1,73 @@
 import { logoutUser, refreshTokens } from 'kitsu/store/auth/actions';
 import store from 'kitsu/store/config';
+import { isNull } from 'lodash';
+import { setToken } from 'kitsu/config/api';
+
+const helper = {
```

Comment:
This name seems a little odd, what makes this a "helper"?  Is there a reason not to just use `let tokenPromise;`?

Reply:
i didn't know if that made it persist in js :/ i'll fix it up